docs/en_US/Tutorial/ExperimentConfig.md

```diff
@@ -130,442 +199,483 @@ machineList:
+If trialGpuNum is bigger than the free gpu numbers, and the trial jobs running simultaneously can not reach __trialConcurrency__ number, some trial jobs will be put into a queue to wait for gpu allocation.
 
-* __versionCheck__
-  * Description
+### maxExecDuration
+
+Optional. String. Default: 999d.
+
+__maxExecDuration__ specifies the max duration time of an experiment. The unit of the time is {__s__, __m__, __h__, __d__}, which means {_seconds_, _minutes_, _hours_, _days_}.
+
+Note: The maxExecDuration spec set the time of an experiment, not a trial job. If the experiment reach the max duration time, the experiment will not stop, but could not submit new trial jobs any more.
+
+### versionCheck
+
+Optional. Bool. Default: false.
   
-    NNI will check the version of nniManager process and the version of trialKeeper in remote, pai and kubernetes platform. If you want to disable version check, you could set versionCheck be false.
+NNI will check the version of nniManager process and the version of trialKeeper in remote, pai and kubernetes platform. If you want to disable version check, you could set versionCheck be false.
+
+### debug
+
+Optional. Bool. Default: false.
+
+Debug mode will set versionCheck to false and set logLevel to be 'debug'.
+
+### maxTrialNum
+
+Optional. Integer between 1 and 99999. Default: 99999.
+
+Specifies the max number of trial jobs created by NNI, including succeeded and failed jobs.
+
+### trainingServicePlatform
 
-* __debug__
-  * Description
+Required. String.
 
-    Debug mode will set versionCheck be False and set logLevel be 'debug'
+Specifies the platform to run the experiment, including __local__, __remote__, __pai__, __kubeflow__, __frameworkcontroller__.
 
-* __maxTrialNum__
-  * Description
+* __local__ run an experiment on local ubuntu machine.
 
-   __maxTrialNum__ specifies the max number of trial jobs created by NNI, including succeeded and failed jobs.
+* __remote__ submit trial jobs to remote ubuntu machines, and __machineList__ field should be filed in order to set up SSH connection to remote machine.
 
-* __trainingServicePlatform__
-  * Description
+* __pai__  submit trial jobs to [OpenPAI](https://github.com/Microsoft/pai) of Microsoft. For more details of pai configuration, please refer to [Guide to PAI Mode](../TrainingService/PaiMode.md)
 
-    __trainingServicePlatform__ specifies the platform to run the experiment, including {__local__, __remote__, __pai__, __kubeflow__}.
+* __kubeflow__ submit trial jobs to [kubeflow](https://www.kubeflow.org/docs/about/kubeflow/), NNI support kubeflow based on normal kubernetes and [azure kubernetes](https://azure.microsoft.com/en-us/services/kubernetes-service/). For detail please refer to [Kubeflow Docs](../TrainingService/KubeflowMode.md)
 
-    * __local__ run an experiment on local ubuntu machine.
+* TODO: explain frameworkcontroller.
```

Comment:
frameworkcontroller submit trial jobs to kubernetes. It is built to orchestrate all kinds of applications (e.g., Tensorflow models, PyTorch models, Python scripts) on Kubernetes by a single controller. For more details of configuration for frameworkcontroller mode, please refer to [FrameworkController Docs]()